stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -211,6 +216,69 @@ public static <ReqT, RespT> Iterator<RespT> blockingServerStreamingCall(
     return result;
   }
 
+  /**
+   * The better way to do server streaming calls.<br>
+   * Call {@link BlockingClientCall#read()} for
+   * retrieving values.  A {@code null} will be returned after the server has closed the stream.
+   * <br>
+   * The methods {@link BlockingClientCall#hasNext()} and {@link
+   * BlockingClientCall#cancel(String, Throwable)} can be used for more extensive control.
+   * <br>
+   * <p> Example usage:</p>
+   * <pre>
+   * {@code  while ((response = call.read()) != null) { ... } }
+   * </pre>
+   * or
+   * <pre>
+   * {@code
+   *   while (call.hasNext()) {
+   *     response = call.read();
+   *     ...
+   *   }
+   * }
+   * </pre>
+   *
+   * @return A {@link BlockingClientCall} that has had the request sent and halfClose called
+   * @throws InterruptedException if it receives an interrupt while sending the request
+   */
+  public static <ReqT, RespT> BlockingClientCall<?, RespT> blockingV2ServerStreamingCall(
+      Channel channel, MethodDescriptor<ReqT, RespT> method, CallOptions callOptions, ReqT req)
+      throws InterruptedException {
+    io.grpc.stub.BlockingClientCall<ReqT, RespT> call =
+        blockingBidiStreamingCall(channel, method, callOptions);
+    call.write(req);
```

Comment:
If interrupted, we need to cancel the RPC. The caller won't receive the `call` object, so won't be able to.